docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -8,13 +8,13 @@ Components of any type can specify the endpoints that the container exposes. The
 
 . Specify endpoints properties as shown in the following example:
 +
-.Specify endpoints properties
+.Specifying endpoints properties
```

Comment:
- `discoverable` is not a property
- `public` is not a property, its `exposure` and it has 3 options - you may refer the schema for desc
- `unsecuredPaths` is not a property
- `cookiesAuthEnabled` is not a property
-